StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1001UnitTests.cs

```diff
@@ -8,6 +8,7 @@ namespace StyleCop.Analyzers.Test.SpacingRules
     using System;
     using System.Threading;
     using System.Threading.Tasks;
+    using Microsoft;
```

Comment:
If you expand the test case that can verify the fix as well, you can remove this using statement at the same time.